src/components/Layout/index.tsx

```diff
@@ -1,9 +1,13 @@
 import { FC, ReactNode } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
+import { ThemeProvider } from 'styled-components';
```

Comment:
Look for the way where you can build `ThemeProvider` without styled-component please